packages/shared/src/components/profile/UpvoterList.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement, memo, ReactNode } from 'react';
+import Link from 'next/link';
+import { Upvote } from '../../graphql/common';
+import { LazyImage } from '../LazyImage';
+
+export interface UpvoterListProps {
+  upvotes: Upvote[];
+  children?: ReactNode;
+}
+
+export function UpvoterList({
+  upvotes,
+  children,
+}: UpvoterListProps): ReactElement {
+  return (
+    <div className="flex flex-col relative">
+      {upvotes.map(({ user }) => (
+        <Link
+          key={user.username}
+          href={`https://app.daily.dev/${user.username}`}
+        >
+          <a className="flex flex-row hover:bg-theme-active px-6 py-3">
+            <LazyImage
+              imgSrc={user.image}
+              imgAlt={user.username}
+              className="w-12 h-12 rounded-10"
+              ratio="1 / 1"
```

Comment:
If you use `w-12 h-12` you already set the dimension of the element so `ratio` is not needed.

Reply:
Strange, it is giving me an element without a height, though the width, after inspecting, is alright. Am I missing something?

![Screen Shot 2021-09-24 at 2 21 03 PM](https://user-images.githubusercontent.com/13744167/134627901-a5ee2668-39ff-472a-91fe-92416868a327.png)